src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/Manifest.cs

```diff
@@ -114,7 +115,7 @@ public static Manifest ReadFrom(Stream stream, Func<string, string> propertyProv
             XDocument document;
             if (propertyProvider == null)
             {
-                document = XmlUtility.LoadSafe(stream, ignoreWhiteSpace: true);
+                document = XmlUtility.Load(stream);
```

Comment:
I guess we need that preserve: ignoreWhiteSpace: true
So better change to: `XmlUtility.Load(path, LoadOptions.PreserveWhitespace);`

Reply:
`ignoreWhiteSpace` is set to `true` by default [here](https://github.com/NuGet/NuGet.Client/blob/dev-kartheekp-ms-shared-xmlutility/build/Shared/XmlUtility.cs#L84-L85) unless an overloaded `Load` method is invoked passing additional argument `LoadOptions.PreserveWhitespace`.

Reply:
Ok.